script/main.js

```diff
@@ -64,55 +71,72 @@ async function showNotebook(){
             </div>
             <div class="options">
                 <span>
-                    <i onClick="editPost(this)" class="fas fa-edit"></i>
+                    <i onClick="editNote(this)" class="fas fa-edit"></i>
                     <i onClick="deleteNote(this)" class="fas fa-trash-alt"></i>
                 </span>
             </div>
         </div>
-      `;
-      
-    }
+      `;    
+    }};
 
-}
-async function deleteNote(){
 
-    const response = await fetch(`${APIUrl}/delete`,{
-        method: 'DELETE',
-        headers: {
-            'Content-type':'application/json'
-        },
-        body: JSON.stringify({id: valueInput})
-    })
+async function deleteNote(){
 
-    if (!response.ok){
-        throw new Error('There is a problem.');
-    }
-    const result = await response.json();
-    console.log(`The result is: ${result}`)
+            const response = await fetch(`${APIUrl}/delete`,{
```

Comment:
make sure we merge in the one that is in the main and not this one, because this will bring a merge conflict